cLIMS/organization/static/siteWide/js/show.js

```diff
@@ -99,10 +99,10 @@ $(document).ready(function(){
        });
         $("#unselect").click(function(){
             alert("If nothing selected all experiments will be considered.")
-            $('input:checkbox[name=dcic]').attr('checked',false);
+            $('input:checkbox').attr('checked',false);
```

Comment:
is that just removing some default and nothing else  ?

Reply:
Intially the DCIC html page has just one table with all exps and all checkbox fields had name "dcic".
But now there are multiple tables on this page, consisting of exps for every expset.
In order to have (un)select all for each table of Exp Set, so the names are modified to include Exp Set id ( dcic(expset.id)" like dcic77)

Also there is a button which should unselect al checkboxes, so I modified the code based on the "input type" to access all boxes on the page, and unselect them.